version: "3.3"

services:

  sandbox-python38:
    build:
      context: ..
      dockerfile: ./docker/Dockerfile
    container_name: sandbox-python38
    image: sandbox-python38
    volumes:
      - ../src:/app/src:ro
    ports:
      - "9001:9001"
    restart: on-failure
